docs/user/package-management/history-and-rollback.md

```diff
@@ -3,34 +3,57 @@ title: History and Software Rollback
 summary: History and Software Rollback
 ---
 
-# History and Software Rollback
+# History and software rollback in Solus
 
-Solus provides a history and rollback feature via its package manager, `eopkg`. This feature allows you to see when your repository was last updated, actions such as installation, removal, and upgrading of software.
+Solus offers a feature to view your history of software changes and revert your system to previous versions. 
 
-Our rollback feature allows you to essentially rewind your system's software back to a previous state, assuming you or our repository has the versions in question.
+You can use this feature if you have issues after installing packages or system updates, and you need to return to a working state.
 
-## History
+## History of software changes
 
-You can see the history from `eopkg` by using:
+- To see your history of software changes, open a terminal and execute:
 
-```bash
-eopkg history
-```
+    ```bash
+    eopkg history
+    ```
+
+    The terminal displays a list of all the software changes in your system.
+
+    ![eopkg history output](./eopkg-history-output.png)
 
 ## Rollback
 
-:::info
+:::warning[Important]
 
-If you have a local repository set up, [disable it](https://help.getsol.us/docs/packaging/advanced-config/local-repository#disabling-the-local-solbuild-repository-in-eopkg) before doing rollbacks or you may get an error.
+Reverting software changes only works if:
 
-> Error parsing repository index information. Index file does not exist or is malformed.
+- The Solus repository has the version of the package you need.
+- You have a local copy of the package.
```

Comment:
We might want to make it clear that it's an OR condition and not an AND condition.

Reply:
Done. Thanks for your feedback!